tools/generate_launchfile.sh

```diff
@@ -44,7 +44,7 @@ IFS=";"
 for camera in $CAMERA_CONFIGS; do
 	IFS="," #Inside each camera configuration, the parameters are comma-separated
 	set $camera
-	if [[ $# != 8 ]]; then
+	if [[ $# < 8 ]]; then
```

Comment:
I'm not sure this will work. 

The check for 8 values here is to make sure that each semicolon - separated camera in $CAMERA_CONFIGS has exactly 8 comma-separated values (name, serial number, x, y, z, roll, pitch, yaw). There can be any number of cameras like that.

To pass in an extra argument for yaml file containing the default parameters, I would suggest introducing a new environment variable, since this file is device-specific (and not camera-specific!)

Reply:
@nicovanduijn done